tgui/packages/tgui/interfaces/HelmConsole.js

```diff
@@ -50,12 +50,21 @@ const SharedContent = (props, context) => {
             })} />
         )}
         buttons={(
+        <>
```

Comment:
did you mean to add this?

Reply:
i was getting massive errors without it from code